modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Unbound event handlers were incorrectly running when unbound in a previous event handler #TINY-7436
```

Comment:
```suggestion
- Unbinding an event handler while that event is firing now takes effect immediately #TINY-7436
```

Definitely [nitpick], I just saw `event handler` and `unbound` too many times in that sentence and it made it harder for me to parse.

Reply:
I see your point, however that suggestion sounds more like a `Changed` when we want this to definitely be a `Fixed`. Paging @tylerkelly13 for his thoughts.

Reply:
```suggestion
- Event handlers could not unbind while the event was firing #TINY-7436
```
Maybe? Don't know if I understand the problem being fixed here.

Reply:
```suggestion
- Unbinding an event handler, while that event was firing, did not take effect immediately #TINY-7436
- Binding an event handler, while that event was firing, incorrect took effect immediately #TINY-7436
```
@ashfordneil how does that look?